views/pages/contact.ejs

```diff
@@ -4,7 +4,7 @@
 <section class="hero">
     <div class="container form-container text-center">
         <p>Get in Touch</p>
-        <form action="" id="contact-form">
+        <form action="/api/v1/contact-us" id="contact-form">
```

Comment:
add the method is it a post or get method

Reply:
is it just this change that have 4 commits?

Reply:
1. updated! with Post method
2. those commits are from contact-us API not the page sir